.changelog/4119.yml

```diff
@@ -0,0 +1,4 @@
+changes:
+- description: Updated the **validate** command to not fail when an image is younger than 3 months instead of 3 days.
```

Comment:
```suggestion
- description: Updated the *DO106* error code in the **validate** command to not fail when a docker image is younger than 3 months instead of 3 days.
```